OsmIntegrator.Tests/Helpers/Base/BaseHelper.cs

```diff
@@ -0,0 +1,75 @@
+﻿using Newtonsoft.Json;
+using OsmIntegrator.ApiModels;
+using OsmIntegrator.ApiModels.Auth;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OsmIntegrator.Tests.Helpers.Base
+{
+    public class BaseHelper
+    {
+        protected const string HttpAddressAndPort = "https://localhost:44388/";
+        protected HttpClient _client;
+
+
+        protected async Task<HttpResponseMessage> LoginAdminAsync()
+        {
+            var loginData = new LoginData
+            {
+                Email = "admin@abcd.pl",
```

Comment:
W projekcie nie ma roli admina w wersji MVP. Jest tylko `user`, `editor` oraz `supervisor`. Nie ma co się logować na to konto bo jest całkowicie nietestowane.

Reply:
admin zmieniłem na user1.